swabbie-core/src/test/kotlin/com/netflix/spinnaker/swabbie/ResourceTypeHandlerTest.kt

```diff
@@ -736,34 +732,4 @@ object ResourceTypeHandlerTest {
     verify(notifier, times(0)).notify(any(), any(), any())
     verify(resourceRepository, times(1)).upsert(updatedMarkedResource)
   }
-
-  internal fun workConfiguration(
```

Comment:
pulled this to a common util, `WorkConfigurationTestHelper.generateWorkConfiguration()`